webcrypto/goja.go

```diff
@@ -29,6 +33,34 @@ func exportArrayBuffer(rt *goja.Runtime, v goja.Value) ([]byte, error) {
 	return ab.Bytes(), nil
 }
 
+// traverseObject traverses the given object using the given fields and returns the value
+// at the end of the traversal. It assumes that all the traversed fields are Objects.
+func traverseObject(rt *goja.Runtime, src *goja.Object, fields ...string) (goja.Value, error) {
+	var value goja.Value = src
+	var currentField string
+	var obj *goja.Object
+
+	for idx, field := range fields {
+		if idx > 0 {
+			currentField += "."
+		}
+
+		currentField += field
```

Comment:
This adds quite a bit of allocations for an error message that *hopefully* will never happen. 

This should go "backwards" and build the message when needed.

I am pretty sure `strings.Join(fields[:idx], ",")` has the same end result and can be calculated only once, and only when needed.